Add Optuna Integrated to README.md

Crissman opened the pull request:
Thanks for your interest in Optuna! Here's a button to show that Optuna is integrated into PyKEEN.

mali-git commented:
@mberr Thanks!

> I would furthermore propose to remove
positive_scores = torch.tensor(positive_scores, dtype=torch.float, device=self.device)
negative_scores = torch.tensor(negative_scores, dtype=torch.float, device=self.device)

Yes, this makes sense!

mberr commented:
> @mberr Thanks!
> 
> > I would furthermore propose to remove
> > positive_scores = torch.tensor(positive_scores, dtype=torch.float, device=self.device)
> > negative_scores = torch.tensor(negative_scores, dtype=torch.float, device=self.device)
> 
> Yes, this makes sense!

@mali-git Will you do this change, or should I open another pull request?

mali-git commented:
> > @mberr Thanks!
> > > I would furthermore propose to remove
> > > positive_scores = torch.tensor(positive_scores, dtype=torch.float, device=self.device)
> > > negative_scores = torch.tensor(negative_scores, dtype=torch.float, device=self.device)
> > 
> > 
> > Yes, this makes sense!
> 
> @mali-git Will you do this change, or should I open another pull request?

However you wish :)
If you have it ready please go ahead, othwerwise I will do it at latest till tomorrow.

mberr commented:
> > > @mberr Thanks!
> > > > I would furthermore propose to remove
> > > > positive_scores = torch.tensor(positive_scores, dtype=torch.float, device=self.device)
> > > > negative_scores = torch.tensor(negative_scores, dtype=torch.float, device=self.device)
> > > 
> > > 
> > > Yes, this makes sense!
> > 
> > 
> > @mali-git Will you do this change, or should I open another pull request?
> 
> However you wish :)
> If you have it ready please go ahead, othwerwise I will do it at latest till tomorrow.

cf. #65

Crissman committed:
Add Optuna Integrated to README.md

Thanks for your interest in Optuna! Here's a button to show that Optuna is integrated into PyKEEN.

cthoyt committed:
Add optuna badge to template

cthoyt committed:
Update README.md

[skip ci]

cthoyt committed:
Update README.md

[skip ci]
